### ISSUE TITLE
fiatleak.com "down"

### ISSUE THREAD

**Issue by marceltt** [andrewhodel/fiatleak#8]
The domain name "fiatleak.com" is currently pointing to a namecheap.com parking page.

**Comment by guihknx** [comment]
http://andrewhodel.github.io/fiatleak/

**Comment by andrewhodel** [comment]
Yes I have taken down the domain as it has put too much of a burden on me
personally.

I need to resolve other things in my life before expanding into new areas.

I'm still debating internally exactly what I should do about it, but I
really do have other things I need to work on.

I'm always open to advice if you have any to give.

Thanks,
Andrew

**Comment by marceltt** [comment]
I think a redirect from http://www.fiatleak.com/ to http://andrewhodel.github.io/fiatleak/ would suffice in the meantime.

**Comment by litecoinmafia** [comment]
I've uploaded the code with some minor modifications to http://www.fiatleak.net! Now live!
